docs/hugo/content/_index.md

```diff
@@ -144,63 +189,47 @@ C:\> SET AZURE_CLIENT_SECRET=<your-client-secret>
 {{% /tab %}}
 {{< /tabpane >}}
 
-3. Install [the latest **v2+** Helm chart](https://github.com/Azure/azure-service-operator/tree/main/v2/charts):
+Then create a secret named `aso-credential` in the namespace you'd like to create ASO resources in.
```

Comment:
A note would be helpful here to instruct users about using `aso-credential` _only_.

Reply:
We already cover that in the authentication documentation which I link below on line 232. I've added a "not recommended" to it on the authentication page though so that folks understand they should prefer namespace scoped.